mutant/config/hasta/artic.json

```diff
@@ -43,9 +47,9 @@ if ( params.medaka || params.nanopolish ) {
     // Clinical genomics override
     params {
         // Minimum read length for artic guppyplex
-        min_length = 400
+        min_length = 150
```

Comment:
Will there be an interest in analyzing both samples that were amplified with artic primers as well as those amplified with midnight primers? If only one will be used to begin, doing this will be perfect. 

When we want to be able to vary this, i think it would be good to have use an if statement here. Something like:
```
if ( params.primersArtic ) {
    min_length = 400
}
if ( params.primersMidnight ) {
    min_length = 150
}
```
And then adapt the start command for running nanopore accordingly.

Reply:
As discussed, lets fix support for Artic the day using Artic for nanopore is considered.